lib/acts_as_paranoid/core.rb

```diff
@@ -112,10 +118,14 @@ def destroy
           end
         end
       else
-        destroy!
+        destroy_fully!
       end
     end
 
+    def destroy
+      destroy!
```

Comment:
@zzak If there isn't any blocker, would you like to merge this? :yellow_heart: :green_heart: :heart: :blue_heart: :heart: Thank you!